tldw_Server_API/app/api/v1/endpoints/media/process_videos.py

```
@@ -0,0 +1,273 @@
+        batch_result = await run_video_batch(
+            all_inputs_to_process=all_inputs_to_process,
+            form_data=form_data,
+            current_user=current_user,
+            temp_dir=str(temp_dir_path),
+            temp_path_to_original_name=temp_path_to_original_name,
+            file_handling_errors_structured=file_handling_errors_structured,
+        )
+
+    # --- Determine Final Status Code & Return ---
+    final_error_count = batch_result.get("errors_count", 0)
+    final_success_count = batch_result.get("processed_count", 0)
```

Ruff (0.14.5)

Local variable `final_success_count` is assigned to but never used

Remove assignment to unused variable `final_success_count`

(F841)